extensions/authentication-jwt/tsconfig.json

```diff
@@ -10,30 +10,5 @@
     "src",
     "src/**/*.json"
   ],
-  "references": [
-    {
-      "path": "../../packages/authentication/tsconfig.json"
-    },
-    {
-      "path": "../../packages/boot/tsconfig.json"
-    },
-    {
-      "path": "../../packages/core/tsconfig.json"
-    },
-    {
-      "path": "../../packages/repository/tsconfig.json"
-    },
-    {
-      "path": "../../packages/rest-explorer/tsconfig.json"
-    },
-    {
-      "path": "../../packages/rest/tsconfig.json"
-    },
-    {
-      "path": "../../packages/security/tsconfig.json"
-    },
-    {
-      "path": "../../packages/service-proxy/tsconfig.json"
-    }
-  ]
+  "references": []
```

Comment:
This should not happen. Once you rebase this branch against `upstream/master` and run `npm i` again at root directory, it should be fixed.